LanguageExt.Core/DataTypes/Try/Try.Extensions.cs

```diff
@@ -528,13 +526,13 @@ public static Seq<A> AsEnumerable<A>(this Try<A> self)
         this Try<A> self,
         Func<A, Try<B>> bind,
         Func<A, B, C> project) =>
-            MTry<A>.Inst.Bind<MTry<C>, Try<C>, C>(self,    a =>
-            MTry<B>.Inst.Bind<MTry<C>, Try<C>, C>(bind(a), b =>
-            MTry<C>.Inst.Return(project(a, b))));
+            MTry<A>.Inst.Bind<MTry<C>, Try<C>, C>(self, a =>
+         MTry<B>.Inst.Bind<MTry<C>, Try<C>, C>(bind(a), b =>
```

Comment:
There appears to be some inconsistent indentation here.  Why aren't they lined up any more?

Reply:
I didn't change this method.  I think we are seeing changes that entered by branch when I merged in `master` in order to resolve the conflicts.  The green lines are what `master` looks like now.

Reply:
To address your actual question, the formatting is now partially specified by the `.editorconfig` file.  When auto formatting, sometimes extra white space is allowed to stay.  Other times, extra white space is removed.

Old line 531 begins the method's expression body.  This is a case where extra white space is allowed to stay.  If you move old line 531 to end of line old line 530 and hit "Enter", you will see that Visual Studio would prefer to indent old line 531 exactly as far as old line 530.

In contrast, towards the end of old line 531 and the beginnings of old lines 532 and 533, extra white space was not allowed to stay because the context is now arguments to a method call.  This is one case in which extra white space is not allowed to say.